content/en/docs/kubernetes/operator/troubleshooting/automatic.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Auto-instrumentation
+---
+
+If you're using the [OpenTelemetry Operator](/docs/kubernetes/operator)'s
+[auto-instrumentation](/docs/kubernetes/operator/automatic) capability and
```

Comment:
```suggestion
If you're using the [OpenTelemetry Operator](/docs/kubernetes/operator)'s
capability to inject [auto-instrumentation](/docs/kubernetes/operator/automatic)  and
```

We might need to review the operator docs later for where "automatic instrumentation" is applicable and where we want to say "zero-code instrumentation", but for now I am ok with leaving it like that.

Reply:
I took a quick look at the [Operator's README on GitHub](https://github.com/open-telemetry/opentelemetry-operator?tab=readme-ov-file#opentelemetry-operator-for-kubernetes) and it still says "auto-instrumentation". I'm guessing that there was a recent shift to rename it to "zero-code instrumentation"?